git/foundations_git/introduction_to_git.md

```diff
@@ -20,7 +20,8 @@ Finally, you will set up a project with Git that will serve as a template for yo
 
 For now, let's learn what Git is and why it's so powerful!
 
-### Lesson overview:
+
+### Lesson Overview
```

Comment:
```suggestion
### Lesson overview
```

We want to keep sentence case according to our style guide. 
I removed the extra line above the header. 

You can click `commit suggestion` to directly commit my suggestion if you want. That way you don't ahve to locally edit and push it again

Reply:
Thank you so much!